### ISSUE TITLE
kiosk: file repository not refreshing as expected

### ISSUE THREAD

**Issue by luizaogs** [arch-kiosk/arch-kiosk-office#2622]
I've never noticed this before so am not sure if it's a new thing. I don't think it's due to my internet connection - it's working fine otherwise. 

I keep tagging images in MKAP as JARCE2024 to find them for a publication. But whenever I first add a tag to an image in the edit file dialogue and then try to fetch the stuff for that tag, the new image is not there. Refreshing the file repository using the browser refresh doesn't help either. I keep having to go back to the hub, then back into the repository, where if I search for the tag again the image is finally there.

It has happened at least three times now, so I let you know.

**Comment by urapadmin** [comment]
Your observation is correct. Makes immediate sense to me. I am not re-fetching the existing tags until the page gets fetched again.

**Comment by luizaogs** [comment]
Ah.

**Comment by urapadmin** [comment]
no. Reads like a misunderstanding to me now. It is the list of tags that you get when you start typing in the tag filter field that might not get fetched again until you refresh the file repository in the browser. That should always work, though. Going back to the Hub and then to the File Repository is more or less a refresh. So that worries me. Cache?

However, once an image gets tagged with a new tag, using that tag in the filter field (by typing it in instead of selecting it) must work right away.

**Comment by urapadmin** [comment]
and my local test works. So I might need an example or a more detailed description here

**Comment by luizaogs** [comment]
I can try to reconstruct it based on my description above. This was a while ago so I don't actually remember.

**Comment by luizaogs** [comment]
PVD appears to be down? I don't want to add bogus tags to actual MKAP pictures.

**Comment by urapadmin** [comment]
yes, PVD and URAP need a longish update due to #3075

**Comment by urapadmin** [comment]
PVD at least is back.

**Comment by luizaogs** [comment]
Woo. You can save me from crying over a syllabus for a 200 student class.

**Comment by luizaogs** [comment]
Well, whatever the issue was seems to not exist anymore. I can't reproduce this either.

**Comment by urapadmin** [comment]
those are good bugs :o) Now I have to find something else to get you off that syllabus. 200 Students sounds scary on many a front.

**Comment by luizaogs** [comment]
Yeah ... it's going to suck.

**Comment by urapadmin** [comment]
I won't torture you with euphemisms or glass-half-full-isms. But you have at least one more ticket you can test, and I expect it to be a bit messy: #3112

**Comment by luizaogs** [comment]
Ha, I already got there :) Do you see how desperate I am for a distraction?
